File: app/src/main/java/com/supercasual/fourtop/network/pojo/DataVoting.java
package com.supercasual.fourtop.network.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DataVoting {

    @SerializedName("photos")
    @Expose
    private List<DataImage> images;
    @SerializedName("vote_token")
    @Expose
    private String voteToken;

    public List<DataImage> getImages() {
        return images;
    }

    public void setImages(List<DataImage> images) {
        this.images = images;
    }

    public String getVoteToken() {
        return voteToken;
    }

    public void setVoteToken(String voteToken) {
        this.voteToken = voteToken;
    }
}

File: app/src/main/java/com/emikhalets/voteapp/network/pojo/DataVoting.kt
package com.emikhalets.voteapp.network.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DataVoting {
    @SerializedName("photos")
    @Expose
    var images: List<DataImage>? = null

    @SerializedName("vote_token")
    @Expose
    var voteToken: String? = null
}